tls/s2n_resume.c

```diff
@@ -62,8 +62,9 @@ static int s2n_tls12_serialize_resumption_state(struct s2n_connection *conn, str
     POSIX_GUARD(s2n_stuffer_write_bytes(to, conn->secure.cipher_suite->iana_value, S2N_TLS_CIPHER_SUITE_LEN));
     POSIX_GUARD(s2n_stuffer_write_uint64(to, now));
     POSIX_GUARD(s2n_stuffer_write_bytes(to, conn->secrets.master_secret, S2N_TLS_SECRET_LEN));
+    POSIX_GUARD(s2n_stuffer_write_uint8(to, conn->ems_negotiated));
 
```

Comment:
Should you gate this on S2N_IN_TEST for now?

Reply:
Nope, we want to gate the EMS feature, not the session ticket increasing in size. ems_negotiated will always be 0 for right now.